src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -248,10 +253,13 @@ export const CollectionHeader: FC<Props> = ({ artworks, collection }) => {
         return (
           <header>
             <Flex flexDirection="column">
-              <Box>
+              <Box mt={[0, "-12px"]}>
                 <Background
+                  position={["relative", "absolute"]}
+                  left={["auto", 0]}
+                  width={["auto", 1]}
```

Comment:
This is the gist of the responsive sizing — we use styled-system's responsive props here to make the `Background` element go absolute / flush-left / full-width once at `sm` and above.